platform/security/sl_component/sl_mbedtls_support/inc/threading_alt.h

```diff
@@ -0,0 +1,234 @@
+/**************************************************************************/ /**
```

Comment:
```suggestion
/**************************************************************************
```

Reply:
just to fix the comments

Reply:
Restyler shenanigans. Fixed.

Reply:
If you leave the / /** it will always put it back to the weird format. Restyler cant process that style of comment

Reply:
I see. Fixed now, I think.